adafruit_minimqtt/adafruit_minimqtt.py

```diff
@@ -126,6 +126,7 @@ class MQTT:
     :param int keep_alive: KeepAlive interval between the broker and the MiniMQTT client.
     :param socket socket_pool: A pool of socket resources available for the given radio.
     :param ssl_context: SSL context for long-lived SSL connections.
+    :param bool use_binary: Sets if message is passed as bytearray instead of string to callbacks
```

Comment:
Possibly `use_binary_mode`? 

Are there other libraries that do this functionality that you know of? Do they require a keyword argument, or do they automatically detect the type?

Reply:
I'm used to using MQTT with Arduino and things are a bit different because in C, a `char` and a `byte` array are basically the same things. But in CircuitPython, the message is read in a `bytearray`, and I don't know of a way to check beforehand if the content will be a valid Unicode string or if is just a binary blob. In C you could just interpret the `char` array the way you need. 

I don't know of a similar library that does the same. Like I mentioned above, on the Arduino lib that we are using, the default behavior is to use binary messages and the user handles the conversion.

Example: https://github.com/arduino-libraries/ArduinoMqttClient/blob/master/examples/WiFiSimpleReceiveCallback/WiFiSimpleReceiveCallback.ino#L104
This one is even better that allows you to stream the message and read bigger messages that don't fit in RAM.

I can change the flag to `use_binary_mode` here.

Reply:
Thanks for the explanation, let's have the user handle conversion in user-code.

Please change the flag to `use_binary_mode` to make it more descriptive and tag me for a re-review